scripts/credential_management.py

```diff
@@ -110,19 +94,11 @@ def delete_meta(client, bucket_name: str, site: str) -> bool:
         bucket = f"{args.bucket}-{args.env}"
     if args.create_auth:
         id_str = create_auth(
-            s3_client,
-            bucket,
             args.create_auth[0],
             args.create_auth[1],
             args.create_auth[2],
         )
-        print(f"{id_str} created")
-    elif args.delete_auth:
-        succeeded = delete_auth(s3_client, bucket, args.delete_auth)
-        if succeeded:
-            print(f"Removed {args.delete_auth}")
-        else:
-            print(f"{args.delete_auth} not found")
+        print(id_str)
```

Comment:
:scissors:? Or modify into something more user-oriented, like you had before with `print(f"{id_str} created")`

Reply:
so right now this isn't actually doing any interaction with secrets manager - so this is basically outputting the key/value pair you'd add to the secrets dict. I could suggest something like that as output text.